sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -237,14 +237,6 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       intercept[AnalysisException] {
         sql(s"""LOAD DATA LOCAL INPATH "$incorrectUri" INTO TABLE non_part_table""")
       }
-
-      // Unset default URI Scheme and Authority: throw exception
```

Comment:
Every time we call `newHadoopConf` we get a new `Configuration`. So I think `hiveContext.sparkContext.hadoopConfiguration.unset("fs.default.name")` will not actually unset the value?